src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -28,6 +34,29 @@ public SchemaRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper obje
                 new SchemaRowMapper());
     }
 
+    public EventTypeSchema getSchemaVersion(final String name, final String version)
```

Comment:
Also I usually put the logic related things inside service layer. In that case repository only does fetch of data.